inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/sidebar/RecommendationSidebar.html

```diff
@@ -18,45 +18,54 @@
 -->
 <html xmlns:wicket="http://wicket.apache.org">
 <wicket:panel>
-  <div class="flex-content flex-v-container flex-gutter flex-only-internal-gutter">
-    <div class="flex-content panel panel-default panel-flex">
-      <div class="panel-heading">
-        <h3 class="panel-title"><wicket:message key="recommender"/></h3>
-      </div>
-      <div class="scrolling panel-body">
-        <form wicket:id="form" class="panel panel-default panel-small">
-          <div class="panel-heading">
-            <h3 class="panel-title"><wicket:message key="settings"/></h3>
-          </div>
-          <div class="panel-body">
-            <div class="col-sm-12">
-              <div class="form-horizontal">
-                <div class="form-group" wicket:enclosure="maxPredictions">
-                  <label class="col-sm-6 control-label"><wicket:message key="maxSuggestions"/></label>
-                  <div class="col-sm-6">
-                    <input wicket:id="maxPredictions" type="number" class="form-control"/>
-                  </div>
-                </div>
-                <div class="form-group" wicket:enclosure="showAllPredictions">
-                  <label class="col-sm-6 control-label"><wicket:message key="showAll"/></label>
-                  <div class="col-sm-6">
-                    <div class="checkbox">
-                      <label wicket:for="showAllPredictions">
-                        <input wicket:id="showAllPredictions" type="checkbox"/>
-                      </label>
-                    </div>
-                  </div>  
-                </div>
-              </div>
-            </div>
-          </div>
-          <div class="panel-footer text-right">
-            <input wicket:id="save" type="submit" class="btn btn-primary"  wicket:message="value:apply"/>
-          </div>
-        </form>
-        <div wicket:id="learningCurve" ></div>
-      </div>
-    </div>
-  </div>
+	<div
```

Comment:
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.